src/integration-tests/model-in-image/mii-sample-wrapper/build-model-image.sh

```diff
@@ -109,17 +109,71 @@ dryrun:  ${MODEL_YAML_FILES:+--wdtModel ${MODEL_YAML_FILES}} \\
 dryrun:  ${MODEL_VARIABLE_FILES:+--wdtVariables ${MODEL_VARIABLE_FILES}} \\
 dryrun:  ${MODEL_ARCHIVE_FILES:+--wdtArchive ${MODEL_ARCHIVE_FILES}} \\
 dryrun:  --wdtModelOnly \\
-dryrun:  --wdtDomainType ${WDT_DOMAIN_TYPE}
+dryrun:  --wdtDomainType ${WDT_DOMAIN_TYPE} \\
+dryrun:  --chown oracle:root
```

Comment:
I assume this needs to be conditional on the domain type?
So at line 75, add lines
```
if [ "$WDT_DOMAIN_TYPE" = "WLS" ]; then
  CHOWN_ROOT="--chown:root"
fi
```
and modify line 113 so it looks like:
```
 ${CHOWN_ROOT:+${CHOWN_ROOT}}
```
The ":+" syntax says 'do nothing if CHOWN_ROOT before the :+ is undefined, otherwise substitute the contents after :+.

Reply:
(This is much less code than cutting and pasting they entire dryrun)